spotdl.sh

```diff
@@ -9,6 +9,7 @@ command -v mp3splt >/dev/null 2>&1 || { echo >&2 "mp3splt was not found"; exit 1
 
 # Get Spotify's pulseaudio sink ID
 get_spotify_sink(){
+	sleep 0.2
```

Comment:
Adding a sleep here goes against the use of waiting for the spotify sink to appear (meaning the song has started playing).
In the case where spotify starts but the music takes x seconds to start, not being able (to my knowledge) to know when the music stops other than waiting for its known length, the music would be truncated before it ends. 0.2 does not make a huge difference, but I guess that's unnecessary

Reply:
On my machine it's necessary; otherwise, Spotify doesn't start reliably. Even 0.1 was sometimes too small. You can still leave it out.